chainer/dataset/convert.py

```diff
@@ -106,6 +106,14 @@ def converter():
     If the batch were requested to be converted to ChainerX with such
     converters, :class:`RuntimeError` will be raised.
 
+    .. note::
+
+        Converters using this decorator can't be pickled
+        causing :class:`chainer.training.updaters.MultiprocessParallelUpdater`
+        to fail when the multiprocessing start mode is set to `spawn` or
+        `forkserver`. Should you need to use such feature, please rely on
```

Comment:
nit:
```suggestion
        to fail when the multiprocessing start mode is set to ``'spawn'`` or
        ``'forkserver'``. Should you need to use such feature, please rely on
```

See e.g. https://github.com/chainer/chainer/blob/37cfc76b81f40f6af70530491198a17af8d92587/chainer/functions/loss/softmax_cross_entropy.py#L510-L511